### PR TITLE
Introduce EPS counter metric.

### PR DESCRIPTION
**Pull request by PremyslCerny**
"Integrating" counter. :)

### CODE REVIEW

**Comment by PremyslCerny** [comment]
@ateska Please review.

File: asab/metrics/metrics.py [hunk 1]
```diff
@@ -102,6 +103,33 @@ def rest_get(self):
 		return rest
 
 
+class EPSCounter(Counter):
```
Comment (ateska) [inline asab/metrics/metrics.py:106]: """
Event per Second Counter
"""

... providing a bit of docstring is helpful here.

**Review by ateska — COMMENTED** [review]
I'm not 100% sure that this is a right path but ... let's merge that and see.

**Review by ateska — CHANGES_REQUESTED** [review]
Add a docstring, please

**Pull request merged**